xbmc/settings/lib/SettingsManager.h

```diff
@@ -92,7 +90,7 @@ class CSettingsManager : public ISettingCreator, public ISettingControlCreator,
    \param root XML node
    \return True if the setting values were successfully saved, false otherwise
    */
-  bool Save(TiXmlNode *root) const override;
+  bool Save(TiXmlNode *root) const;
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
This change only removes the `override` keyword. Furthermore a following commit completely changes this line.

Reply:
E.g. if the commit which remove this line gets reverted the alignment will be wrong again.

Reply:
Yes and if I either fix it in the same commit (which IMO is ugly) or in a cleanup commit and the whole PR is reverted it is also wrong again.

I completely agree that we should follow the coding guidelines in newly added code.
But as long as nobody has fixed all the existing code I'm not going to fix the coding style of every line I touch in existing code. That just makes it much more work to edit existing code and it puts the code into a completely messed up state coding style wise because some lines are fixed but most others are not which results in a mix of completely different coding styles throughout a file.